readme.txt

```diff
@@ -1,188 +1,188 @@
-
-= Version 2.1.3 =
-
-* Move the `error_reporting()` call in the AJAX handler to the beginning so that we're more sure that no PHP errors are outputted. Some hosts disable usage of `set_time_limit()` and calling it was causing a PHP warning to be outputted.
-
-= Version 2.1.2 =
-
-* When regenerating all images, newest images are done first rather than the oldest.
-* Fixed a bug with regeneration error reporting in some browsers. Thanks to pete-sch for reporting the error.
-* Supress PHP errors in the AJAX handler to avoid sending an invalid JSON response. Thanks to pete-sch for reporting the error.
-* Better and more detailed error reporting for when `wp_generate_attachment_metadata()` fails.
-
-= Version 2.1.1 =
-
-* Clean up the wording a bit to better match the new features and just be easier to understand.
-* Updated screenshots.
-
-= Version 2.1.0 =
-
-Lots of new features!
-
-* Thanks to a lot of jQuery help from [Boris Schapira](http://borisschapira.com/), a failed image regeneration will no longer stop the whole process.
-* The results of each image regeneration is now outputted. You can easily see which images were successfully regenerated and which failed. Was inspired by a concept by Boris.
-* There is now a button on the regeneration page that will allow you to abort resizing images for any reason. Based on code by Boris.
-* You can now regenerate single images from the Media page. The link to do so will show up in the actions list when you hover over the row.
-* You can now bulk regenerate multiple from the Media page. Check the boxes and then select "Regenerate Thumbnails" form the "Bulk Actions" dropdown. WordPress 3.1+ only.
-* The total time that the regeneration process took is now displayed in the final status message.
-* jQuery UI Progressbar version upgraded.
-
-= Version 2.0.3 =
-
-* Switch out deprecated function call.
-
-= Version 2.0.2 =
-
-* Directly query the database to only fetch what the plugin needs (the attachment ID). This will reduce the memory required as it's not storing the whole row for each attachment.
-
-= Version 2.0.1 =
-
-* I accidentally left a `check_admin_referer()` (nonce check) commented out.
-
-= Version 2.0.0 =
-
-* Recoded from scratch. Now uses an AJAX request per attachment to do the resizing. No more PHP maximum execution time errors or anything like that. Also features a pretty progress bar to let the user know how it's going.
-
-= Version 1.1.0 =
-
-* WordPress 2.7 updates -- code + UI. Thanks to jdub and Patrick F.
-
-= Version 1.0.0 =
-
-* Initial release.
-
-= Upgrade Notice =
-Support for WordPress 5.3
+=== Regenerate Thumbnails ===
+Contributors: Viper007Bond
+Tags: thumbnail, thumbnails, post thumbnail, post thumbnails
+Requires at least: 5.3
+Tested up to: 5.5
+Requires PHP: 7.0

```

Comment:
Is there a strong reason to bump these numbers as a part of this PR? Seems like something we may want to evaluate separately, given the large userbase for this plugin.

Reply:
It was to make it easier to address the failing tests (i.e. not having to fix tests on really old versions). It can be pulled into a separate PR if you like.

Reply:
I've moved the test and version bump changes to https://github.com/Automattic/regenerate-thumbnails/pull/112